core/src/autogluon/core/models/ensemble/bagged_ensemble_model.py

```diff
@@ -252,6 +252,8 @@ def predict_proba(self, X, normalize=None, **kwargs):
 
         if self.temperature_scalar is not None:
             pred_proba = self._apply_temperature_scaling(pred_proba)
+        elif self.conformalize is not None:
+            pred_proba = self._apply_conformalization(pred_proba)
```

Comment:
above in AbstractModel you apply `_apply_conformalization` to `y_pred` but here to `pred_proba`. Is this an issue, in particular if we later try to extend conformalization to classification as well?

Reply:
I just move the `_apply_conformalization` to be applied to `y_pred_proba` in AbstractModel, so this align with the code in BaggedEnsembleModel.